charts/pulsar/templates/pulsar-cluster-initialize.yaml

```
@@ -114,5 +114,9 @@ spec:
       volumes:
       {{- include "pulsar.toolset.certs.volumes" . | nindent 6 }}
       restartPolicy: OnFailure
+      {{- if .Values.pulsar_metadata.nodeSelector }}
+      nodeSelector:
+{{ toYaml .Values.pulsar_metadata.nodeSelector | indent 6 }}
```

I am pretty sure this needs to be indented 8, not 6. Were you able to verify this change works?

```suggestion
{{ toYaml .Values.pulsar_metadata.nodeSelector | indent 8 }}
```